File: Game/java/Item.java
/*
 * Item class
 */

public class Item {

	private String type;
	private String description;
	private boolean status;

	public Item(String type, String description, boolean status){
    	this.type = type;
    	this.description = description;
    	this.status = status;
    }



    /*
    * Getters
    */
    public String getType(){
    	return type;
    }

    public String getDesc(){
    	return description;
    }

    public boolean getStatus(){
    	return status;
    }

    /*
    * Setters
    */
    public void setDescription(String newDesc){
    	description = newDesc;
    }

    public void setStatus(boolean newStatus){
    	status = newStatus;
    }

} 

File: Game/js-attempt/public/item.js
class Item {
  constructor(type, description, status) {
    this.type = type;
    this.description = description;
    this.status = status;
  }

  get type() {
    return this.type;
  }

  get description() {
    return this.description;
  }

  get status() {
    return this.status;
  }

  set type(newType) {
    return this.type = newType;
  }

  set description(newDesc) {
    return this.description = newDesc;
  }

  set status(newStatus) {
    return this.status = newStatus;
  }

}